### PR TITLE
Core 1611 Fix Analysis Landing Page infinite loop

### PR DESCRIPTION
**Pull request by sri2k1us**
SSIA although am not entirely sure what was causing this infinite loop of `History` and `stat` service call. When I removed the duplicate check of  `isInfoFetching`, it fixed the issue. 😕

### CODE REVIEW

File: src/components/analyses/landing/AnalysisSubmissionLanding.js [hunk 1]
```diff
@@ -132,38 +132,32 @@ export default function AnalysisSubmissionLanding(props) {
...
+                    setAnalysis({
+                        ...analysis,
+                        resultfolderid,
+                        status,
+                        enddate,
+                    });
+                }
+            }
+        }
+    }, [analysis, currentNotification, id]);
```
Comment (sri2k1us) [inline src/components/analyses/landing/AnalysisSubmissionLanding.js:160]: Mostly a refactor cos `React.useCallback` or having the logic inside `useEffect` is the same.

File: src/serviceFacades/analyses.js [hunk 2]
```diff
@@ -283,7 +282,6 @@ function useAnalysisParameters({ id, enabled, onSuccess, onError }) {
         onSuccess: (data) => {
             onSuccess(preProcessData(data));
         },
-        onError,
```
Comment (sri2k1us) [inline src/serviceFacades/analyses.js:286]: remove unused params.

**Review by psarando — APPROVED** [review]
LGTM 👍 

Though I don't really spot the fix either; all I can think of is the callback may have been causing issues? 🤷

**Comment by sri2k1us** [comment]
Thanks for the reviews.

**Pull request merged**